cloud-premium: add manual backup support for premium instances (#22615)
pingcap/docs: tidb-cloud/premium/backup-and-restore-premium.md

2. Locate the corresponding backup file you want to delete, and click **...** > **Delete** in the **Action** column.

## Manual backups

In addition to automatic backups, {{{ .premium }}} supports manual backups. Manual backups provide a user-controlled, guaranteed restore point, which is highly recommended before performing high-risk actions such as system upgrades, critical data deletion, and irreversible schema or configuration changes.

> ### Key characteristics of manual backups:

Reviewer: ```suggestion
### Key characteristics
```

- **Retention and deletion**: unlike automatic backups, manual backups are not automatically deleted based on retention rules. They are retained indefinitely until you explicitly delete them. If the instance is deleted, its manual backups are moved to the Recycle Bin and will remain there permanently until manual deletion.

- **Storage location**: manual backups are stored in cloud storage managed by TiDB.

- **Cost**: due to their long-term retention, manual backups are subject to additional charges.

> - **Limitations**: manual backups do not support Point-in-Time Recovery (PITR) or partial backups (for example, table-level or database-level). Restoring a manual backup into an existing or running instance is not supported. Each restore requires a new instance.

Reviewer: ```suggestion
- **Limitations**: manual backups do not support point-in-time recovery (PITR) or partial backups (for example, table-level or database-level backups). You cannot restore a manual backup to an existing instance. Each restore operation creates a new instance.
```

> - **Permissions**: both **Organization owners** and **Instance managers** can perform manual backups. However, only **Organization owners** can perform restore actions for system-managed manual backups.

Reviewer: ```suggestion
- **Permissions**: both `Organization Owner` and `Instance Manager` can create manual backups. Only `Organization Owner` can restore system-managed manual backups.
```

### Create a manual backup

1. Navigate to the [**Backup**](#view-the-backup-page) page of your instance.

2. In the upper-right corner, click **...**, and then click **Manual Backup**.

3. Confirm the operation. The backup is stored in TiDB Cloud and will appear in the **Backup List**. 

> You can restore the manual backup directly through the UI without requiring external storage credentials.

Reviewer: ```suggestion
You can restore a manual backup directly in the TiDB Cloud console without providing external storage credentials.
```

TiDB Cloud supports snapshot restore and point-in-time restore for your instance.

> - **Snapshot Restore**: restores your instance from a specific backup snapshot. Both automatic and manual backups can be restored this way. Manual backups are displayed in the Backup List with a "Manual" backup type and a "Permanent" expiration status.

Reviewer 2: ```suggestion
- **Snapshot Restore**: restores your instance from a specific backup snapshot. Both automatic and manual backups can be restored this way. Manual backups are displayed in the **Backup List** with a **Manual** backup type and a **Permanent** expiration status.
```
